js/main.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
+
+var Node = function(data){
+  this.data = data;
+  this.next = null;
+  this.previous = null;
+}
+
+var List = function(){
+  this.head = null;
+  this.tail = null;
+  this.length = 0;
+}
+
+List.prototype.getHead = function(){
+  return this.head;
+}
+
+List.prototype.getTail = function(){
+  return this.tail;
+}
+
+List.prototype.append = function(data){
+  var node = new Node(data);
+
+  if(this.length){
+    this.tail.next = node;
+    node.previous = this.tail;
+    this.tail = node;
+  }
+  else{
+    this.head = node;
+    this.tail = node;
+  }
+
+  this.length++;
+
+  return node;
+}
+
+List.prototype.prepend = function(data){
+  var node = new Node(data);
+
+  if(this.length){
+    this.head.previous = node;
+    node.next = this.head;
+    this.head = node;
+  }
+  else{
+    this.head = node;
+    this.tail = node;
+  }
+
+  this.length++;
+
+  return node;
+}
+
+List.prototype.at = function(position){
```

Comment:
What will happen if the position is incorrect value? (for example, nonexistent)

Reply:
okey, i will handle exeption